### DISCUSSION TITLE
Docker image latest tag version

### DISCUSSION THREAD

**Discussion by craig-seeman** [chaos-mesh/chaos-mesh#3691] [General]
I'm a bit curious what others thoughts are on the `:latest` tag that is being advertised for the docker images. I could see some possible end-user confusion by using `:latest` and getting new / (possibly) buggy / different features than what is in the official documentation.

Would a better approach possibly be to have :latest tag to the current production release version as opposed to the latest build of the master branch? At the same time, offering a `:latest-dev` for the master branch?

Thanks!

**Comment by STRRL** [comment]
We should use the certain versioned docker image when we release the Chaos Mesh, for example, Chaos Mesh v2.4.1 would use these images:

- `ghcr.io/chaos-mesh/chaos-mesh:v2.4.1`
- `ghcr.io/chaos-mesh/chaos-daemon:v2.4.1`
- `ghcr.io/chaos-mesh/chaos-dashboard:v2.4.1`

and the `latest` tag is only used for the master branch, which is not ready for production.

If you see some released versions using the docker image with the `latest` version, that's **definitely a bug**, please report it to us!  Thanks! ❤️
